### PR TITLE
ensure that ReportAppConfigs always have uuids

### PR DESCRIPTION
**Pull request by nickpell**
@orangejenny

### CODE REVIEW

**Comment by orangejenny** [comment]
looks good pending tests

File: corehq/apps/app_manager/models.py [hunk 1]
```diff
@@ -3303,15 +3304,14 @@ class ReportAppConfig(DocumentSchema):
...
 
-    @classmethod
-    def wrap(cls, obj):
-        # todo: can remove once existing ReportAppConfig have been migrated
-        _soft_assert_uuid(obj.get('uuid'), 'Check uuids for apps containing UCR %s' % obj.get('report_id'))
-        return super(ReportAppConfig, cls).wrap(obj)
+    def __init__(self, *args, **kwargs):
+        super(ReportAppConfig, self).__init__(*args, **kwargs)
+        if not self.uuid:
+            self.uuid = random_hex()
```
Comment (czue) [inline corehq/apps/app_manager/models.py]: how do you feel about adding a few simple tests? 

1. make a new one and ensure valid ID
2. wrap an existing one and ensure same ID

(am sure this works but that seems quick and nice to have)

Comment (czue) [inline corehq/apps/app_manager/models.py]: and I think the reason I'm asking is that `__init__` functions of db models are pretty big hotspots that we should make sure are well tested

Comment (nickpell) [inline corehq/apps/app_manager/models.py]: :+1:

Comment (nickpell) [inline corehq/apps/app_manager/models.py]: added tests

Comment (czue) [inline corehq/apps/app_manager/models.py]: looks good thanks!

**Pull request merged**